pyproject.toml

```diff
@@ -21,6 +21,7 @@ dependencies = [
     "termcolor >= 1.0.0",
     "textx >= 2.2",
     "astunparse >= 1.6.0", # astunparse is only needed for Python3.8, we should use ast.unparse when we drop Python3.8.
+    "packaging",
```

Comment:
Where is this module used?

Reply:
Here:
https://github.com/pyccel/pyccel/blob/f9753b6e87a815dd8d34d7280d3e7db65e5ab350/pyccel/commands/lambdify.py#L6-L14

Reply:
I see! Thanks